lib-driver/caqti_driver_mariadb.mli

```diff
@@ -12,14 +12,16 @@
  *
  * You should have received a copy of the GNU Lesser General Public License
  * along with this library.  If not, see <http://www.gnu.org/licenses/>.
- *)
+*)
```

Comment:
Please don't remove the indentation of end-of-comment tokens.

Reply:
Ah, sorry about that.  I just accepted whatever ocp-indent gave me in this case.